authorization-server/package.json

```diff
@@ -30,6 +30,11 @@
     "test:e2e": "cd test && jest --config ./jest-e2e.json"
   },
   "dependencies": {
+    "@energyweb/credential-governance": "^1.0.1-alpha.242.0",
+    "@ew-did-registry/did": "^0.7.0",
+    "@ew-did-registry/did-ethr-resolver": "^0.7.0",
+    "@ew-did-registry/did-ipfs-store": "0.6.3-alpha.710.0",
```

Comment:
```suggestion
    "@ew-did-registry/did-ipfs-store": "^0.7.0",
```

Reply:
I needed to downgrade to this version otherwise, the TS compiler throws errors.

Reply:
If you revert a0abd0e395fdc3a4e6db39483ff0eacf9261226e commit, code does to compile.